### PR TITLE
User can start a course

### PR DESCRIPTION
**Pull request by mukeshm**
PR for issue #8

### CODE REVIEW

File: src/components/CreateCourse.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      if (this.state.name === '') {
+        this.setState({nameClass: 'inputText errorInput'})
+      }
+      if (this.state.startdate === '') {
+        this.setState({dateClass: 'inputText errorInput'})
+      }
+      if (this.state.duration === null || this.state.duration <= 0) {
+        this.setState({durationClass: 'inputText errorInput'})
+      }
+    }else {
```
Comment (deobald) [inline src/components/CreateCourse.js]: The above code is pretty repetitive and a lot of nested ifs. How do you think you could make it cleaner? :)

**Review by deobald — CHANGES_REQUESTED** [review]
Nice job! I only had one comment to make.

File: src/components/CreateCourse.js (hunk 1 above)
Comment (rsbh) [reply src/components/CreateCourse.js]: ok, will try to make it cleaner.

Comment (rsbh) [reply src/components/CreateCourse.js]: @deobald All the nested ifs and repetitive code are removed.

**Review by deobald — COMMENTED** [review]
wups! I forgot to submit the review. sorry.

**Pull request merged**